### ISSUE TITLE
make admins & timezones required

### ISSUE THREAD

**Issue by nilsnolde** [valhalla/valhalla#4467]
we talked about it and came to the conclusion that there's no point to have timezones & admins optional. there's tons of issues with no admins added, and also having no timezones results in strange behavior if someone requests with time. @kevinkreiser suggested we could host those somewhere so people can pull them without building. we could use github artifacts for that, triggered by a manual github action to re-create them. those artifacts are available for download via the github api. though they're only persisted for 90 days or so I think, but it's easy to automatically generate them every couple of months, github actions has cron capabilities.

could even auto-download them in the code eventually via a config url defaulting to the latest github artifact, but the whole HTTP/curl compilation logic seems a bit daunting currently..

**Comment by kevinkreiser** [comment]
i was thinking of having them be part of a "release" so that they persist forever. it is often the case for admins that you basically dont want to touch them if you finally found a version of the planet where all of them seem unbroken :smile: for timezones it changes infrequently so the same applies but for a different reason

**Comment by kevinkreiser** [comment]
this here where we can attach binaries to a release: 
![image](https://github.com/valhalla/valhalla/assets/697548/dee299d0-837f-4e75-95d0-0c01c290db40)

**Comment by nilsnolde** [comment]
Ah also cool actually. Then we’d ideally release every time we know admin stuff or timezone stuff changed (not counting broken osm shit of course). that’ll be on a tighter schedule than our current release habits;)

**Comment by kevinkreiser** [comment]
hahahaha sick burn :smile:

**Comment by ianthetechie** [comment]
> it is often the case for admins that you basically dont want to touch them if you finally found a version of the planet where all of them seem unbroken

Yeah, I'm with @kevinkreiser on this one ;) in practice I update admins _maybe_ once/month tops and have never had any issues / heard complaints from end users.

Given that fact, I have it on my TODO list to look at using [Daylight](https://daylightmap.org/) for admins. I'll report back what I find (someone poke me please if I don't update within a few weeks), but it seems rather appealing for this use case, as it 1) will be _reasonably_ up to date, 2) is completely OSM data, but 3) ostensibly tends to be "less broken" on average.

**Comment by nilsnolde** [comment]
Don’t we even parse ALL of the admins, despite only using level 2 & 4? I’ve never seen a broken country, and I check regularly. States maybe in some places? Even if, that’s hardly relevant. In other words, I’d not be too worried about broken admins either for Valhalla. For geocoding/other purposes that’s a totally different discussion.

**Comment by ianthetechie** [comment]
> Don’t we even parse ALL of the admins, despite only using level 2 & 4?

Quite possibly...

> I’ve never seen a broken country, and I check regularly. States maybe in some places? Even if, that’s hardly relevant. In other words, I’d not be too worried about broken admins either for Valhalla. 

I've never had anyone complain with our hosted services either, but that may be a function of it being heavily skewed towards NA and EU which have pretty good data and large communities.

The main reason I think Daylight / Overture are interesting is that it's purely OSM derived, on a rolling snapshot + patches basis (and is also fairly certain to have any vandalism of admin names reverted, though then again that has also been fairly rare; it's more commonly an issue with city and other place names). Those are interesting properties (IMO) for published "base" datasets from the Valhalla project. Just a thought.

**Comment by nilsnolde** [comment]
Yeah sorry, I should’ve made it clearer that this is my personal, heavily biased opinion against those distros. You’re very likely right regarding admins in Daylight. In fact I think it’s the only thing it’s useful for in real life.

**Comment by kevinkreiser** [comment]
it's also very useful if you're worried about buildings coverage in your map display since it's has a bunch of added sources for that

**Comment by nilsnolde** [comment]
aren't the additional footprints AI generated? in my book that'd be barely usable. seems they add some ESRI data, but sounds like only for addressing, not footprints. admittedly their footprint source description [is just 2 sentences](https://daylightmap.org/buildings.html).
